content/_index.md

```diff
@@ -54,7 +54,7 @@ Veřejná správa pravidelně investuje do nových digitálních technologií, d
 
 ## Digitální transformace není úspěšná díky nedostupnosti nebo nedostatečnému využití dat
 
-_Státní správa a valná většina veřejných institucí dostatečně vepracuje s vlastními daty, nepracuje mnohdy ani s jinými daty. Ať už jde o tvorbu veřejných politik postavených na informovaném vládnutí nebo automatizaci agendy, stát ve svém úsilí zaostává. Politiky jsou častěji vytvářeny na základě pravidel politického PR a bez doložených faktů a poznání. Vládnutí je i nadále netransparentní a neadresné. K automatizaci dochází jen velmi pomalu a zpravidla úspěšně jen tam, kde je k technologiím blíže z podstaty typu agendy dané instituce, ve které se tak děje._
+_Politiky jsou častěji vytvářeny na základě pravidel politického PR a bez doložených faktů a poznání. Vládnutí je i nadále netransparentní a neadresné. K automatizaci dochází jen velmi pomalu a zpravidla úspěšně jen tam, kde je k technologiím blíže z podstaty typu agendy dané instituce, ve které se tak děje._
```

Comment:
@jsfsmd Josefe, priznam se, ze bych do toho netahala politiky a politicke PR. To prohlaseni je dost silne. Ponechala bych naopak tu cast v uvodu,

Reply:
politické PR ale není sprosté slovo :) politiky jsou důvod, proč ty rezorty existují, data potřebujeme pro to, abychom měli politiky postavené na faktech a důkazech na jedné straně a na straně druhé také k tomu, abychom automatizovaly procesy - tak jsem to myslel. Na doslovném znění netrvám, jde mi zejména o pointu.

Reply:
Nechapu to negativne, jen me prislo, ze to "bez dolozenych faktu a poznani" bylo prilis ostre. Ale upravit muzeme vzdycky, pointa je jasna.